### PR TITLE
fix issue 295, solve number types problems

### LINKED ISSUE #295: Long values being generated as Int
Seeing values that were unsigned integers in C# being designated as `Int` types.  They should have been `Long` types.  Doesn't seem like all instances of this are incorrect but a significant number are.

### PR DESCRIPTION
**Pull request by xurui1995**
fix issue 295, solve number types problems fix #295

### CODE REVIEW

File: src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt [hunk 1]
```diff
@@ -184,6 +184,37 @@ fun theSamePrimitiveType(first: JsonPrimitive, second: JsonPrimitive): Boolean {
...
 }
 
+
+/**
+ * Return the kotlin type of a JsonArray with all elements are numbers
+ */
+fun getKotlinNumberClass(jsonArray: JsonArray) : KotlinClass {
+    var ans : KotlinClass =  KotlinClass.INT
+    jsonArray.forEach {
+        if (it.isJsonPrimitive.not() || it.asJsonPrimitive.isNumber.not()) {
```
Comment (wuseal) [inline src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt:194]: Unused code?

**Review by wuseal — APPROVED** [review]
LGTM

File: src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt (hunk 1 above)
Comment (xurui1995) [reply src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt:194]: @wuseal  it‘s used in GenericListClassGeneratorByJSONArray.kt  and ListClassGeneratorByJSONArray.kt

Comment (wuseal) [reply src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt:194]: image
I mean here

Comment (xurui1995) [reply src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt:194]: just added

**Pull request merged**